backend/src/apiserver/resource/client_manager_fake.go

```diff
@@ -76,6 +78,7 @@ func NewFakeClientManager(time util.TimeInterface, uuid util.UUIDGeneratorInterf
 		objectStore:                 storage.NewFakeObjectStore(),
 		scheduledWorkflowClientFake: NewScheduledWorkflowClientFake(),
 		podClientFake:               FakePodClient{},
+		kfamClientFake:              client.NewKFAMClient("", ""),
```

Comment:
This should be a fake client with test logic.

Reply:
Done.